docs/DeveloperGuide.adoc

```diff
@@ -840,20 +1018,76 @@ Use case ends.
 * 3a. The given index is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. TuitionCor shows an error message.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+[discrete]
+=== Use case: Sort student's list by locality
+
+*MSS*
+
+1.  User types “/sort sl” in to the command line.
+2.  TuitionCor will return a list of students or tutors based sorted by their locality.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. System detects an invalid input.
++
+[none]
+** 1a1. TuitionCor will display invalid command or display sort message usage.
+** 1a2. User enter required information.
+** 1a3. Steps 3a1 and 3a2 are repeated until valid input is entered.
++
+Use case resumes at step 2.
+
+[none]
+* 1b. System detects no further input.
++
+Use case ends.
+
+
+[discrete]
+=== Use case: Match
+
+*MSS*
+
+1.  User enter match followed by a person name.
+2.  TuitionCor displays information of client in the respective list (either tutor list or student list)
+3.  TuitionCor returns a list of potential matches in the respective list.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. Client cannot be found.
++
+Use case ends.
+
+* 3a. There is no match found.
++
+[none]
+** 3a1. TuitionCor returns an empty list.
++
+Use case ends.
+
```

Comment:
Where are the other use cases for your enhancements? This is also part of documentation.  @Zhu-Jiahui @shookshire @olimhc